dawn/src/driver-includes/to_json.cpp

```diff
@@ -0,0 +1,77 @@
+#include "to_json.hpp"
+
+using namespace dawn;
+
+MetricsSerialiser::MetricsSerialiser(VerificationMetrics metricsStruct, std::string metricsPath, std::string stencilName,
+                                     std::string fieldIdentifier) {
+        fieldId = fieldIdentifier;
+        path = metricsPath;
+        stencil = stencilName;
+        newJsonMetrics = generateJsonFromStruct(metricsStruct);
+    }
```

Comment:
In C++ we use initializer lists for contructors. In this case this would look like this:
```
MetricsSerialiser::MetricsSerialiser(VerificationMetrics metricsStruct, std::string metricsPath, std::string stencilName,
                                     std::string fieldIdentifier) : fieldId(fieldIdentifier), path(metricsPath), stencil(stencilName), 
                                     newJsonMetrics(generateJsonFromStruct(metricsStruct))
                                     {}
```

Reply:
Thanks that's good to know.